packages/react-notifications/src/components/notification-kinds/api-error/api-error.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import {
+  TAppNotification,
+  TAppNotificationValuesApiError,
+  NOTIFICATION_DOMAINS,
+  NOTIFICATION_KINDS_PAGE,
+} from '@commercetools-frontend/constants';
+import Notification from '../../notification';
+import ApiErrorMessage from '../api-error-message';
+
+type Props = {
+  notification: TAppNotification<{
+    domain: typeof NOTIFICATION_DOMAINS.PAGE;
+    kind: typeof NOTIFICATION_KINDS_PAGE['api-error'];
+    values: TAppNotificationValuesApiError;
```

Comment:
The conditional typings of `TAppNotification` turned out to be pretty nice, as we can specify the exact shape of the notification.